rubric/rubric_peer-review.md

```diff
@@ -17,7 +17,7 @@ This rubric is for assessing one student's peer review of another student's work
 	<td>The review is so generic that it's hard to determine which assignment it's reviewing. It doesn't offer any praise and yet doesn't offer any suggestions either.</td>
 	<td></td>
 	<td></td>
```

Comment:
And

_The review is generally constructive and contains valuable points for improvement. It might not be necessarily well organized, to the point, and concise._

for this one?